### ISSUE TITLE
Alternative to react-native-collapsible

### ISSUE THREAD

**Issue by slorber** [wcandillon/can-it-be-done-in-react-native#65]
Hey @wcandillon, your Accordion looks great.

If ever you plan to opensource a reanimated alternative to https://github.com/oblador/react-native-collapsible that would be so great.

I think there's a real need for that. I can't stand anymore the non-native height animation, it's "ok" but for DEV on Android it's really painfully slow. Unfortunately, don't have much time to work on this atm.

If anyone here is willing to implement this, I'd be happy to be a beta tester

**Comment by wcandillon** [comment]
I agree. Unfortunately, I won't have the capacity to maintain such a package. But I would love it if this example would help to bootstrap something.

I should also mention that here the tap gesture would drop some frame on low-grade Android devices but we could use the TapGestureHandler for that so we would have the entire execution on the native thread.

**Comment by dsznajder** [comment]
@slorber If this is needed, I can build and maintain something on reanimated Transition API, as this should not be some "magic" for everyone. Also, checkout 
https://github.com/shariqahmed1/react-native-animated-tree-view project. If you need some specific animated tree transitions.

**Comment by slorber** [comment]
Hi

I'm definitively interested to have an equivalent of collapsible, and could provide help. 

Your project tree view looks great but it's too much for my usecase: I just want a view that animates from height 0 to height auto. I can manage the other animations (like dropdown arrow rotation etc) myself. Unfortunately didn't have time yet to check what's possible with the transition apis for now
